src/main/native/cpp/URCL.cpp

```diff
@@ -8,36 +8,37 @@
 #include "URCL.h"
 #include "URCLDriver.h"
 
+#include <cstring>
+#include <frc/DataLogManager.h>
 #include <frc/Errors.h>
 #include <frc/Notifier.h>
+#include <functional>
+#include <iostream>
 #include <networktables/NetworkTableInstance.h>
 #include <networktables/RawTopic.h>
-#include <wpi/DataLog.h>
-#include <units/time.h>
-#include <cstring>
-#include <string>
 #include <sstream>
 #include <stdlib.h>
-#include <functional>
-#include <iostream>
+#include <string>
+#include <string_view>
+#include <units/time.h>
+#include <wpi/DataLog.h>
 
 static constexpr auto period = 20_ms;
 
 bool URCL::running = false;
-char* URCL::persistentBuffer = nullptr;
-char* URCL::periodicBuffer = nullptr;
-nt::RawPublisher URCL::persistentPublisher = 
-  nt::NetworkTableInstance::GetDefault()
-    .GetRawTopic("/URCL/Raw/Persistent")
-    .Publish("URCLr2_persistent");
-nt::RawPublisher URCL::periodicPublisher = 
-  nt::NetworkTableInstance::GetDefault()
-    .GetRawTopic("/URCL/Raw/Periodic")
-    .Publish("URCLr2_periodic");
-nt::RawPublisher URCL::aliasesPublisher = 
-  nt::NetworkTableInstance::GetDefault()
-    .GetRawTopic("/URCL/Raw/Aliases")
-    .Publish("URCLr2_aliases");
+char *URCL::persistentBuffer = nullptr;
+char *URCL::periodicBuffer = nullptr;
+nt::RawPublisher URCL::persistentPublisher =
+    nt::NetworkTableInstance::GetDefault()
+        .GetRawTopic("/URCL/Raw/Persistent")
+        .Publish("URCLr2_persistent");
+nt::RawPublisher URCL::periodicPublisher =
+    nt::NetworkTableInstance::GetDefault()
+        .GetRawTopic("/URCL/Raw/Periodic")
+        .Publish("URCLr2_periodic");
+nt::RawPublisher URCL::aliasesPublisher = nt::NetworkTableInstance::GetDefault()
+                                              .GetRawTopic("/URCL/Raw/Aliases")
+                                              .Publish("URCLr2_aliases");
```

Comment:
It seems like this is duplicated from where the publishers are initialized in `Start`